### DISCUSSION TITLE
Shorten the Meta Template for Angular?

### DISCUSSION THREAD

**Discussion by fireflysemantics** [storybookjs/storybook#24510] [Ideas]
### Is your feature request related to a problem? Please describe.

I'm curious if it's possible to shorten the `Meta` declaration for Standalone Angular component stories.

Here's an example declaration:

```
const meta: Meta<ElFlipContainerStoryComponent> = {
  title: 'Flip Container/ Material Cards',
  component: ElFlipContainerStoryComponent,
  tags: ['autodocs'],
  render: (args: ElFlipContainerStoryComponent) => ({
    props: {
      ...args,
    },
  }),
  decorators: [
    moduleMetadata({
      imports: [ElFlipContainerStoryComponent],
      declarations: [],
    }),
  ],
};
```

And it seems the `render` and `decorators` blocks are redundant (Unless they are customized more) since the  generic type `ElFlipContainerStoryComponent` is all the information needed to create these blocks.

### Describe the solution you'd like

It would be nice if all we had to do is declare this.
```
const meta: Meta<ElFlipContainerStoryComponent> = {
  title: 'Flip Container/ Material Cards',
  tags: ['autodocs'],
};
```

And Storybook would use the generic type`ElFlipContainerStoryComponent` to construct the rest of the `Meta` object as needed.

### Describe alternatives you've considered

Just filling in the whole thing for now ...

### Are you able to assist to bring the feature to reality?

no

**Comment by valentinpalkovic** [comment]
Since Typescript is compiled down to Javascript, there aren't any type information during runtime. Therefore, the `component` field is strictly necessary. And as far as I know, in this simple case `render` and `decorators` shouldn't be needed.
